test/NUITizenGallery/TimePickerTest1.py

```diff
@@ -0,0 +1,190 @@
+# test python script must be in same location as aurum_pb2.py
+
+import grpc
+from aurum_pb2 import *
+from aurum_pb2_grpc import BootstrapStub
+from NUIGalleryTestUtils import *
+import time
+
+isTimePickerPageOpened = False
+XPos = 0
+YPos = 0
+
+
+# Check if switch test page is opened or not.
+def CheckTimePickerTestStart(stub):
+    if not LaunchAppTest(stub):
+        return False
+
+    global isTimePickerPageOpened
+    isTimePickerPageOpened = FindTCByInputText(stub, "TimePickerTest1")
+    time.sleep(1)
+    return isTimePickerPageOpened
+
+
+# Check Set
+def CheckTimePickerTest1(stub):
+    if not isTimePickerPageOpened:
+        return False
+
+    res = stub.findElements(ReqFindElements(widgetType='Button'))
+    for elem in res.elements:
+        if "Set" in elem.text:
+            print("elem : ", elem)
+            stub.click(ReqClick(type="ELEMENTID", elementId=elem.elementId))
+            time.sleep(1)
+
+            GetGeometryInfo(stub)
+
+            stub.flick(ReqFlick(startPoint=Point(x=XPos + 90, y=YPos + 205), endPoint=Point(x=XPos + 90, y=YPos + 50), durationMs=110))

```

Comment:
I still do not recommand to use integer here. we can get width and height so I think it would better to be
```suggestion
stub.flick(ReqFlick(startPoint=Point(x=XPos + Width/2, y=YPos + (Height/5 * 4)), endPoint=Point(x=XPos + Width/2, y=YPos + Height/5), durationMs=110))
```

Reply:
Hi, Mr. Lee, 
I update the code, 
and I have tested on two different targets, it works well, please check it, thanks.